addons/web/static/src/views/fields/boolean_toggle/boolean_toggle_field.js

```diff
@@ -10,6 +10,21 @@ export class BooleanToggleField extends BooleanField {
     get isReadonly() {
         return this.props.record.isReadonly(this.props.name);
     }
+
+    async onChange(newValue) {
+        await this.props.record.update({ [this.props.name]: newValue });
+        if (this.props.record.selected && this.props.record.model.multiEdit) {
+            return;
+        }
+        const rootRecord =
+            this.props.record.model.root instanceof this.props.record.constructor &&
+            this.props.record.model.root;
```

Comment:
```suggestion
                        this.props.record.model.root &&
 this.props.record.model.root instanceof this.props.record.constructor
```

Reply:
instanceof or equal ?

Reply:
This is a copy/paste of the code deleted from addons/web/static/src/views/fields/field.js

I supposed it was correct.